cnf-certification-test/platform/suite.go

```diff
@@ -185,16 +185,24 @@ func testTainted(env *provider.TestEnvironment, testerFuncs nodetainted.TaintedF
 		moduleTaintsFound := false
 		otherTaintsFound := false
 
+		otherTaints := []string{}
 		for _, it := range individualTaints {
 			if strings.Contains(it, `module was loaded`) {
 				moduleTaintsFound = true
 			} else {
 				otherTaintsFound = true
+				otherTaints = append(otherTaints, it)
 			}
 		}
 
 		if otherTaintsFound {
 			nodeTaintsAccepted = false
+
+			// Surface more information about tainted kernel failures that have nothing to do with modules.
+			tnf.ClaimFilePrintf("Please note that taints other than 'module was loaded' were found.")
```

Comment:
also add a similar line in the claim file for the tainted modules, see line 213:
logrus.Debug("Collected all of the tainted modules: ", taintedModules)

Reply:
I've run manually this test case and I saw we're not printing the name of the node we're testing... may you add it to the traces, please? I don't think the name of the debug pod is actually helpful for the customer.